sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -48,8 +48,8 @@ Method call will now require an 'id' field to be present in the document body.
 > for more details on consistency levels, or the README section on this change [here](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/cosmos/azure-cosmos#note-on-client-consistency-levels).
 
 #### Features Added
-- Added new **provisional** `max_integrated_cache_staleness_in_ms` parameter to read item and query items APIs in order
-  to make use of the **preview** CosmosDB integrated cache functionality.
+- Added new `max_integrated_cache_staleness_in_ms` parameter to read item and query items APIs in order
```

Comment:
Instead of editing these entries, just make a new one for the new release that says GA of integrated cache functionality